cmd/assetsvc/postgresql_utils.go

```diff
@@ -31,6 +31,9 @@ import (
 // TODO(mnelson): standardise error API for package.
 var ErrChartVersionNotFound = errors.New("chart version not found")
 
+// Temporal flag
```

Comment:
Seems odd that I'm commenting on a temporary comment for a temporary flag, but I get the impression that you like to learn and improve in everything you do :) I think you mean "temporary" as opposed to  "permanent", rather than "temporal" as opposed to "spatial". See https://english.stackexchange.com/questions/7539/temporal-vs-temporary

Reply:
Hahaha, thank you for your comment! :) False friends are often a bit tricky. In Spanish, `temporal` does mean both things (temporal and temporary)